modules/sdk/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplates.java

```diff
@@ -329,6 +329,7 @@ private void _checkArgs(ProjectTemplatesArgs projectTemplatesArgs) {
 			className += "Activator";
 		}
 		else if ((template.equals("mvc-portlet") ||
+				  template.equals("freemarker-portlet") ||
```

Comment:
we should probably make this smarter somehow so we aren't having to check manually, what do you think @Ithildir